height: 50.0
width: 50.0
cell_size: 0.4
DEBUG: true
floor_filter: true
heightmap_threshold: 0.15
heightmap_cell_size: 0.25
max_height: 2.5